src/main/kotlin/app/revanced/patches/twitch/misc/integrations/patch/TwitchIntegrationsPatch.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.twitch.misc.integrations.patch
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patches.twitch.misc.integrations.fingerprints.InitFingerprint
+import app.revanced.patches.twitch.misc.integrations.annotations.TwitchIntegrationsCompatibility
+import app.revanced.shared.patches.AbstractIntegrationsPatch
+
+@Name("twitch-integrations")
```

Comment:
The application name can be omitted here because patches are already scoped to packages implicitely.

```suggestion
@Name("integrations")
```